### PR TITLE
added identity service

### PR DESCRIPTION
**Pull request by rcladmin**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/identityservice/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+### Azure AKS Cluster
+
+Create an AKS Linux cluster in the Azure portal or add Identity Service to an existing AKS Linux cluster. For detailed instructions on setting up an AKS cluster refer to the  [Azure AKS Documentation](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
+
+In the cluster, install the following if they are not already installed :
+
+* NGINX ingress controller
+* Helm
+* KUBE-LEO
```
Comment (unguiculus) [inline incubator/identityservice/README.md]: - Helm is a prerequisite anyways. No need to specify it here.
- Why is nginx-ingress a requirement? Can't you use any other ingress controller?
- kube-lego is deprecated. It shouldn't be required either.

File: incubator/identityservice/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+### SendGrid Email Service
+
+Identity Service requires verification of emails when a user first registers. We use the SendGrid email service for this. Install SendGrid in your Azure account or use an existing installation and copy the **SendGrid API Key** for use later on. For detailed instructions, refer to the  [Azure Send Grid Documentation](https://docs.microsoft.com/en-us/azure/sendgrid-dotnet-how-to-send-email)
+
+### Helm repository
+
+Add the RCL Helm repository on your development system if it does not exist.
+
+```bash
+helm repo add rcl-apps https://rcladmin.github.io/helm-charts/
```
Comment (unguiculus) [inline incubator/identityservice/README.md]: Wrong repo.

File: incubator/identityservice/README.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+Once Identity Service is running in your cluster, you can access the UI dashboard at the following address :
+
+https://your-dns-name/identitysvc
+
+You will need to do some configuration of the UI before you can use the application. Just follow the steps on the setup page when you first open the UI dashboard.
+
+### Helm chart configuration parameters
+
+Please avoid commas (,) in your parameters.
+
```
Comment (unguiculus) [inline incubator/identityservice/README.md]: Add all params to readme

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please make sure you follow our best practices. Compare with what you get from `helm create` and update accordingly.

File: incubator/identityservice/README.md [hunk 4]
```diff
@@ -85,24 +84,12 @@ Identity Service requires verification of emails when a user first registers. We
 
 ### Helm repository
```
Comment (unguiculus) [inline incubator/identityservice/README.md]: Remove line

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
Please fix your CLA

File: incubator/identityservice/README.md (hunk 4 above)
Comment (rcladmin) [reply incubator/identityservice/README.md]: made change. How do I regain cla?

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please compare with what you get from `helm create` using Helm 2.8+ and update accordingly.

Also, please study our review guidelines and apply best practices: https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

**Pull request closed without merging**